core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -271,10 +280,52 @@ public void clicked(InputEvent event, float x, float y) {
         });
         mainStage.addActor(exitButton);
 
+        settingsButton = new TextButton("Settings",skin);
+        settingsButton.setPosition(Config.APPLICATION_WIDTH/2-Config.APPLICATION_WIDTH/15,Config.APPLICATION_HEIGHT/2-(startGameButton.getHeight()+BUTTON_OFFSET));
+        settingsButton.addListener(new ClickListener(){
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                hideAllActors(mainStage);
+                menuShowing = false;
+                settingsShowing = true;
+            }
+        });
+        mainStage.addActor(settingsButton);
+
+        gameMaster.setGameOver(false);
+        GlobalGameState.resetMoves();
```

Comment:
Create an own reset method and call this method at the "Back to Menu" button event.